<!-- only navsat -->

<launch>
    <!-- NAVSAT TRANSFORM NODE -->
    <node pkg="nostop_robot_localization" type="navsat_transform_node" name="navsat_node_roomba" respawn="true">

      <param name="magnetic_declination_radians" value="0"/>
      <param name="yaw_offset" value="0"/>
      <param name="use_odometry_yaw" value="true"/>
      <param name="zero_altitude" value="true"/>
      <param name="wait_for_datum" value="false"/>
      <param name="delay" value="0"/>
      <param name="map_frame" value="SRworld"/>
      <param name="odom_frame" value="red_blue/odom"/>
      <param name="base_link_frame" value="red_blue/base_link"/>
      <param name="world_frame" value="SRworld"/>
      <param name="debug" value="true"/>
       <!--  Placeholder for input topic remapping-->
       <!--remap from="don't touch" to="my_topic" /-->
      <remap from="/imu/data" to="/red_blue/imu_data" />
      <remap from="/gps/fix" to="/red_blue/localizer/gps/fix" />
      <remap from="/odometry/filtered" to="/roomba/red_blue/local_ekf_output" />
      
       <!-- Placeholders for output remapping. -->
      <remap from="/odometry/gps" to="/roomba/red_blue/odom_from_gps"/>
      <remap from="/gps/filtered" to="/gps_filtered"/> 
    </node>
</launch>